pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Pull Requests
+
+Please be sure you have read our documentation regarding contributing
+here: https://docs.pulpproject.org/index.html
+
+There you can read a Contribution Checklist and learn more about
+appropriate branches and rebasing among other things.
+
+Once you are ready to create a pull request:
+
+- [ ] If there is a bug for this issue, please title the pull request
+"<bug_id> - Short Message"
```

Comment:
Hmm we typically don't put the bug id in our PR title.

Reply:
Agreed, but that is what our docs says to do.  We need to pick one and be consistent.

Reply:
I think thats what our Pulp 2 docs say to do, not Pulp 3.